website_version_ce/models/ir_qweb.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-#
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+"""
+Website-context rendering needs to add some metadata to rendered fields,
+as well as render a few fields differently.
+
+Also, adds methods to convert values back to openerp models.
+"""
+
+from openerp import models
+
+
+class QWeb(models.AbstractModel):
+    """ QWeb object for rendering stuff in the website context
+    """
+    _inherit = 'ir.qweb'
+
+    def render(self, cr, uid, id_or_xml_id, qwebcontext=None, loader=None,
```

Comment:
I'm unsure about this one, since that would make the id_or_xml_id assume the "ids" position, which is not what is used with this function: a single id or xml_id is used instead.

Despite searching, I've yet to find an example of redefinition with the new api.
I did find this issue raised on what appears to be an odoo-based project: http://gitlab.opengest.solutions/opengest/odoo/commit/e1deae7eab8be77d7bcb46a68560b86ffbb2277a?view=parallel
The issue suggests that trying to override the function with the new API causes context-related problems.